.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12-dev']
```

Comment:
it should be `'3.12'` as  Python 3.12.0 was released :D

Reply:
```suggestion
        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
```